src/components/Map.ts

```diff
@@ -223,17 +225,20 @@ export class Map extends Component<MapProps, MapState> {
                             location.longitude = Number(locationLookup.lng);
                             this.setState({ locations: props.locations });
                             this.updateBounds(props, location);
+                        } else {
+                            invalidLocations.push(`${location.address}`);
                         }
                     });
                 } else if (this.validLocation(location)) {
                     this.updateBounds(props, location);
                 }
             });
+            if (invalidLocations.length > 0) {
+                this.setState({ alertMessage: `Can not find address ${invalidLocations.join(", ")}` });
```

Comment:
This message is showing, still when loading.... the address is still resolving (#222) a second later and it disappears.

Reply:
Fix. The was an implementation by JS that keeps a stack of invalid location then it shows an error. The implementation has been removed because it does not work given the processing location is async.